specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,551 @@
+# App Notifications in Windows App SDK
```

Comment:
in email we talked about the new toast display abilities of Action Center, stacked notifications, etc. those changes have been back ported to Windows 10. some apps will need to detect this. should that be reflected in the API?

Reply:
DISCUSSION: This will be addressed in a future iteration of this design.